ui/appui-react/src/appui-react/UiFramework.ts

```diff
@@ -614,15 +614,15 @@ export class UiFramework {
     UiFramework.dispatchActionToStore(ConfigurableUiActionId.UseToolAsToolSettingsLabel, value, true);
   }
 
-  /** @alpha */
+  /** @public */
```

Comment:
Just so user dont have to search the setter method to understand the difference.
```suggestion
  /** When `true`, panels will close as soon as the mouse leave the panel.
   * When `false` (default), panels will close on next click outside the panel.
   * @public */
```